src/database/migrations/20220813005900_create_profiles_table.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = (knex) => knex.schema.createTable('profiles', (table) => {
+  table.increments('id');
+
+  table.string('name').notNullable();
+  table.string('username').notNullable();
+  table.string('password').notNullable();
+  table.string('avatar_url').nullable();
```

Comment:
What's avatar_url?

Reply:
It's a reference to the profile avatar on the server. We don't usually save images on the database, but instead we store them in a `/public` directory on the server and point to it on the front-end.

When we develop the profile creation, we'll upload the avatar as a file and store it on the server, but only the URL for reference will be saved on the database.

Reply:
Lgtm

Reply:
Nice, now i understand.
Thank you!